Sources/SharedServices/DynamoUserRepository.swift

```diff
@@ -87,9 +87,14 @@ struct DynamoUserRepository {
 
     func linkGithub(with discordId: String, _ githubId: String) async throws -> String {
         // TODO: Wip
-        guard let discordUser = try await getUser(discord: discordId) else {
-            fatalError()
-        }
+        let query = DynamoDB.QueryInput(
+            expressionAttributeValues: [":v1": .s("DISCORD-\(discordId)")],
+            indexName: discordIndex,
+            keyConditionExpression: "discordID = :v1",
```

Comment:
still need to use `data1` unfortunately.
This query is basically equal to something like Fluent's 
```swift
DynamoDBUser.query(on: db)
.filter(\.$discordID == "DISCORD-\(discordId)")
.all()
```
The problem is, `DynamoDBUser` doesn't have `\.$discordID`, and for now only has `\.$data1`.